lib/src/generate/options.dart

```diff
@@ -57,9 +57,11 @@ class BuildOptions {
       } else {
         color = _red;
       }
-      var message = '$color[${record.level}]$_endColor ${record.loggerName}: '
+      var message = '${Platform.isWindows ? '' : '\x1b[2K\r'}'
```

Comment:
Oh ha, it deletes the entire previous line and sets the cursor back to the beginning. Seems to be the easiest way.